tools/crowtty/src/main.rs

```diff
@@ -63,9 +63,15 @@ struct Args {
     #[arg(short, long, global = true)]
     verbose: bool,
 
-    /// maximum `tracing` level to request from the target.
-    #[arg(short, long, global = true, default_value_t = LevelFilter::INFO)]
-    trace_level: LevelFilter,
+    /// a comma-separated list of `tracing` targets and levels to enable.
```

Comment:
It might be nice to have an example of how to use this in the CLI output, or maybe in the crowtty readme (does crowtty have a readme?)? Maybe something like `--trace=kernel=log,maitake=trace,postcard::flavors=debug` or whatever.

Reply:
crowtty has basically no docs lol...we should fix that.

Reply:
https://github.com/tosc-rs/mnemos/pull/156/commits/f5b52500034a90db9df8ac1483d2f1a86a35a88d adds a quick description of how this works.